### PR TITLE
optimize: skipper route separation lifo to fifo

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
We have to wait for https://github.com/zalando/skipper/pull/2073

**Comment by szuecs** [comment]
:+1:

**Comment by demonCoder95** [comment]
👍

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -136,6 +136,7 @@ spec:
           - "-metrics-flavour=prometheus"
           - "-enable-connection-metrics"
           - "-enable-route-lifo-metrics"
```
Comment (demonCoder95) [inline cluster/manifests/skipper/deployment.yaml:138]: Are we keeping the `lifo` metrics on purpose?

Comment (demonCoder95) [reply cluster/manifests/skipper/deployment.yaml:138]: Keeping it until the rollout of fifo is complete.

Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml:138]: Yes as long as we have lifo somewhere

**Pull request merged**